putio.py

```diff
@@ -321,7 +321,7 @@ def get(cls, id):
     @classmethod
     def add_url(cls, url, parent_id=0, extract=False, callback_url=None):
         d = cls.client.request('/transfers/add', method='POST', data=dict(
-            url=url, save_parent_id=parent_id, extract=extract,
+            url=url, parent_id=parent_id, extract=extract,
```

Comment:
```/transfers/add``` accepts ```save_parent_id```, not ```parent_id```. We should not change it.

See docs: https://api.put.io/v2/docs/transfers.html#add